sqlmesh/core/snapshot/definition.py

```diff
@@ -771,6 +771,9 @@ def missing_intervals(
         Returns:
             A list of all the missing intervals as epoch timestamps.
         """
+        # If the node says that it has an end, and we are wanting to load past it, then we can return no empty intervals
+        if self.node.end and to_datetime(start) > to_datetime(self.node.end):
```

Comment:
Shouldn't this be `>=`?

Reply:
This test would fail if it was like that (why I added the test): https://github.com/TobikoData/sqlmesh/blob/bbe357ea48fa4ccb83f1ca6e1a12900eb9514097/tests/core/test_scheduler.py#L278-L280